website/docs/pact_nirvana/step_4.md

```diff
@@ -23,56 +29,52 @@ There are two reasons that a verification task will need to be run:
   * The verification task will run as part of the normal CI/CD pipeline for the provider
   * The verification task will fetch and verify all the relevant pacts from all consumers from the Pact Broker to ensure no regressions have occurred.
 2. When the pact changes
-  * We recommend that a separate pipeline is set up to verify just the changed pact (more details on this below).
+  * We recommend that a separate pipeline is set up to verify just the changed pact (more details on this later in the guide).
 
 ### Consumer pipeline
 
-The following diagram shows what a fully fledged Pact supported release pipeline might look like for a consumer. During this step of the set up guide, we are just going to cover publishing the pact, and configuring the webhook. The other parts of this pipeline will be covered in later steps.
+The following diagram shows what a fully fledged Pact supported release pipeline might look like for a consumer. During this step of the set up guide, we are just going to cover publishing the pact. The other parts of this pipeline will be covered in later steps.
 
-![Consumer release pipeline](/img/advanced-pact-workshop-diagrams-consumer-pipeline.png)
 
-### Provider pipeline
 
-The following diagram shows what a fully fledged Pact supported release pipeline might look like for a provider. During this step of the set up guide, we are just going to cover the pact verification and results publication. The other parts of this pipeline will be covered in later steps.
+<details open>
+  <summary>Consumer release pipeline</summary>
 
-![Provider release pipeline](/img/advanced-pact-workshop-diagrams-provider-pipeline.png)
+![Consumer release pipeline](/img/advanced-pact-workshop-diagrams-consumer-pipeline.png)
```

Comment:
Suggestion(trivial): it might make sense to try and align the way these UML diagrams are generated (compare them to the ones in the Introduction section, and even there they differ). It currently feels a little weird seeing all these different notations and colors. But, again, this is a trivial low value suggestion.